pkg/test/expectations/expectations.go

```diff
@@ -27,3 +29,24 @@ import (
 func ExpectUnavailable(env *test.Environment, instanceType string, zone string, capacityType string) {
 	Expect(env.UnavailableOfferingsCache.IsUnavailable(instanceType, fmt.Sprintf("%s-%s", fake.Region, zone), capacityType)).To(BeTrue())
 }
+
+func ExpectKubeletFlags(env *test.Environment, customData string, expectedFlags map[string]string) {
+	kubeletFlags := customData[strings.Index(customData, "KUBELET_FLAGS=")+len("KUBELET_FLAGS="):]
```

Comment:
Consider this (using "KUBELET_NODE_LABELS" is a little fagile, but avoids sucking in the rest of custom data):
```suggestion
	kubeletFlags := customData[strings.Index(customData, "KUBELET_FLAGS=")+len("KUBELET_FLAGS=") : strings.Index(customData, "KUBELET_NODE_LABELS")]
```

Reply:
Also, custom expectations should normally have `GinkoHelper()` as first line.
(And we can consider co-locating them the suites they are using in, if they are only used in one.)

Reply:
Really seems like we should at some point refactor all of suite_test for instancetypes to other places rather than putting all of those tests there. 

So im going to wait on co-locating them and leave them here. 

But i did take your suggestion on KUBELET_NODE_LABELS